### PR TITLE
add tests for transform object

### PR DESCRIPTION
**Pull request by chrkhl**
Issue #144

Adds tests for transform object.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: __tests__/transform/object.test.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              "value": "16",
+              "comment": "the base size of the font"
+            },
+            "large": {
+              "value": "20",
+              "comment": "the large size of the font"
+            }
+          }
+        },
+        "color": "#FFFF00"
```
Comment (didoo) [inline __tests__/transform/object.test.js]: I think this should point to an object with a "value" key, like the other "tokens", or if you want to test that it outputs a property as is, maybe just use a "test" key with a "this is a test" string as value

File: __tests__/transform/object.test.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      };
+
+      const options = {
```
Comment (didoo) [inline __tests__/transform/object.test.js]: is this a duplicate of the declaration above?

File: __tests__/transform/object.test.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  ]
+};
+
+describe('transform', () => {
+  describe('object', () => {
+    it('does not crash when called without parameters', () => {
+      expect(transformObject()).toEqual({});
+    })
+
+    it('returns expected object', () => {
```
Comment (didoo) [inline __tests__/transform/object.test.js]: what do you think of splitting this test in smaller tests to check the different cases that I see in the code for `lib/transform/object` ("is plain object and has value", "is just plain object", "else")? no strong opinions on this, just an idea

Comment (dbanksdesign) [reply __tests__/transform/object.test.js]: I think smaller tests are better, they will be easier to debug if a test breaks. Really each test should test 1 thing.

File: __tests__/transform/object.test.js (hunk 1 above)
Comment (dbanksdesign) [reply __tests__/transform/object.test.js]: @didoo  is correct.

**Comment by chrkhl** [comment]
Hey @didoo @dbanksdesign ,
thanks a lot for your feedback. I adopted your proposed improvements.

**Comment by chazzmoney** [comment]
LGTM!

**Pull request merged**